lib/cli/src/commands/run.rs

```diff
@@ -645,17 +645,20 @@ impl Run {
     }
 }
 
-fn start_spinner(msg: String) -> Option<spinner::SpinnerHandle> {
+fn start_spinner(msg: String) -> Option<spinoff::Spinner> {
     if !isatty::stdout_isatty() {
         return None;
     }
-    Some(
-        spinner::SpinnerBuilder::new(msg)
-            .spinner(vec![
-                "⣾", "⣽", "⣻", "⢿", "⡿", "⣟", "⣯", "⣷", " ", "⠁", "⠂", "⠄", "⡀", "⢀", "⠠", "⠐", "⠈",
-            ])
-            .start(),
-    )
+    #[cfg(target_os = "windows")]
+    {
+        use colored::control;
+        let _ = control::enable_virtual_terminal(true);
+    }
```

Comment:
Should we be calling [`colored::control::set_virtual_terminal()`](https://docs.rs/colored/2.0.0/x86_64-pc-windows-msvc/colored/control/fn.set_virtual_terminal.html) here?

Reply:
According to the docs, yes: https://github.com/ad4mx/spinoff#note-for-windows-users